app/components/blacklight/skip_link_component.rb

```diff
@@ -3,7 +3,8 @@
 module Blacklight
   class SkipLinkComponent < Blacklight::Component
     def link_to_search
-      link_to t('blacklight.skip_links.search_field'), '#search_field', class: link_classes
+      search_id = helpers.blacklight_config.search_fields.many? ? '#search_field' : '#q'
```

Comment:
I'm not sure it's enough to check whether multiple fields are configured; don't we need to reproduce the rest of the logic that's used in the SearchBarComponent?

https://github.com/projectblacklight/blacklight/blob/main/app/components/blacklight/search_bar_component.html.erb#L13

I think it'd be something like:
```suggestion
      search_id = helpers.blacklight_config.search_fields.values.select { |field_def| helpers.should_render_field?(field_def) }.many? ? '#search_field' : '#q'
```

(but maybe it'd be cleaner to extract `search_id` to its own method or something?